app/components/Blockchain/Transaction/ClaimAbstract.jsx

```diff
@@ -53,7 +57,16 @@ export default class ClaimAbstract extends React.Component<Props> {
               }
             </Fragment>
           </div>
-          <div className={styles.historyButtonPlaceholder} />
+          {
```

Comment:
I don't think there is a need to wrap the `Button` with the curly braces

Reply:
Whoops, thank you!